packages/unmock-node/src/fs-service-def-loader.ts

```diff
@@ -1,9 +1,10 @@
 import debug from "debug";
 import * as fs from "fs";
+import * as _ from "lodash";
```

Comment:
where is `lodash` used here?

Reply:
Removed, linter was confused as comments included underscore.